pycqed/instrument_drivers/physical_instruments/ZurichInstruments/ZI_HDAWG8.py

```diff
@@ -447,19 +481,35 @@ def upload_waveform_realtime(self, w0, w1, awg_nr: int, wf_nr: int =1):
         self._realtime_w0 = w0
         self._realtime_w1 = w1
 
+        print('wf_nr', wf_nr)
+        print('w0', len(w0))
+        print('w1', len(w1))
+
         # Checked and everything matches
```

Comment:
Commented out code like this should be removed.

Reply:
Done